v2/guides/blocks/architecture-with-sdk.mdx

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
not related to this file, but we should have some way of pre generating all the possible combinations and their content and stored on git in this repo (they dont have to be served on the frontend). This would be useful to see the affect of making changes. For example, if i make a change to a multi tenancy writeup, i would want to make sure that change is only applied to the relavant set of generated pages. So seeing a git diff of the actual resulting guides would be very helpful